CONTRIBUTING.md

```diff
@@ -1,40 +1,145 @@
 # Contributing
 
-Contributions are a vital part of this library and imgix's commitment to open-source. We welcome all contributions which align with this project's goals. All we ask is that you please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
+Thank you for investing your time in contributing to this project! Please take a moment to review this document in order to streamline the contribution process for you and any reviewers involved.
 
-Following these guidelines helps to communicate that you respect the time of
-the developers managing and developing this open source project. In return,
-they should reciprocate that respect in addressing your issue or assessing
-patches and features.
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR
 
 <!-- NB: Run `npx markdown-toc CONTRIBUTING.md` to generate TOC :) -->
 
 <!-- prettier-ignore-start -->
 
 - [Contributing](#contributing)
-  * [Project Goals](#project-goals)
-  * [Using the issue tracker](#using-the-issue-tracker)
-  * [Development](#development)
-  * [Code Conventions](#code-conventions)
-    + [Project setup](#project-setup)
-    + [Compiles and hot-reloads for development](#compiles-and-hot-reloads-for-development)
-    + [Compiles and minifies for production](#compiles-and-minifies-for-production)
-    + [Run your unit tests](#run-your-unit-tests)
-    + [Run your end-to-end tests](#run-your-end-to-end-tests)
-    + [Lints and fixes files](#lints-and-fixes-files)
-    + [Customize configuration](#customize-configuration)
+  - [New contributor guide](#new-contributor-guide)
+  - [Opening a Pull Request](#opening-a-pull-request)
+    - [Checklist](#checklist)
+  - [Conventional Commit Spec](#conventional-commit-spec)
+  - [Example workflow](#example-workflow)
+- [Project Goals](#project-goals)
+- [Using the issue tracker](#using-the-issue-tracker)
+- [Development](#development)
+- [Publishing](#publishing)
+- [Code Conventions](#code-conventions)
+- [Project setup](#project-setup)
+  - [Compiles and hot-reloads for development](#compiles-and-hot-reloads-for-development)
+  - [Compiles and minifies for production](#compiles-and-minifies-for-production)
+  - [Run your unit tests](#run-your-unit-tests)
+  - [Run your end-to-end tests](#run-your-end-to-end-tests)
+  - [Lints and fixes files](#lints-and-fixes-files)
+  - [Customize configuration](#customize-configuration)
+
+<!-- prettier-ignore-end -->.
```

Comment:
I think we can do away with the TOC section here since GH supports native TOC for markdown files now.

Reply:
Agreed, left it in since it was there before but happy to take it out 👌🏼